### ISSUE TITLE
Feature request: Continuous text without compiling

### ISSUE THREAD

**Issue by hamishlow** [kevboh/longform#23]
The Longform plugin is fantastic!  Thanks so much Kevin!
One thing that would make it perfect would be if the scenes could be viewed continuously, much like in Scrivener.  
I am dragging notes into the Draft folder, to rearrange and write and it would be amazing if I could scroll down, and see note pane after note pane of the entire text of every scene arranged vertically while still preserving the ability to rearrange these notes and insert other notes.  I could order then compile the scenes and then edit the draft as a single note, but then I lose some of the advantages of being able to adjust and rearrange which I have when the scenes are separate.

**Comment by kevboh** [comment]
Interesting! I suspect this may go beyond the limits of what an Obsidian plugin can do, but it's a good idea--I'll take a look. Glad you like the plugin.

**Comment by hamishlow** [comment]
Yes, https://github.com/gasparschott/obsidian-continuous-mode basically achieves this, and can apply to the Longform view if those scenes are open...
